Service/MoveService.cs

```diff
@@ -8,14 +8,17 @@ public class MoveService
     {
         private readonly IMoveRepository _moveRepository;
         private readonly IImageRepository _imageRepository;
+        private readonly ImageService _imageService;
```

Comment:
Powinieneś wstrzyknąć interfejs, zrób interfejs w którym masz definicje metod które są w serwisie, zarejestruj to jako AddScoped w Startup.cs  i tutaj wstrzyknij interfejs - główna zasada Dependency Injection